src/documentation/system_administrators/advanced/quantum_safe_storage/quantum_safe_storage_deploy.md

```diff
@@ -0,0 +1,594 @@
+
+As before, check the output for any failures to retrieve data. You can retry them in the same way:
+
+```sh
+# Note that we skipped the line with [[groups]]
+for node in $back end_NODES; do
+  # The rest of the script is the same
+  # ...
+```
+
+Once every data back end has a valid entry in the config file, we are done with this section of the deployment.
+
+### Storing Zstor Config
+
+The `zstor-default.toml` file contains sensitive information that is sufficient to recover and decrypt all of the data stored in your QSFS. Needless to say, you should keep the contents of this file secure.
+
+If your front end machine is lost for any reason, the zstor config file can be used to recover the data. On the other hand, if the contents of this file are lost, the data in the back ends can never be recovered.
+
+*Consider storing the entire contents of your `zstor-default.toml` file in a durable and secure data store like a password manager.*
+
+## Frontend System Prep
+
+At this point, we are ready to proceed the setup of the frontend system. We'll demonstrate all necessary commands to do this in any Linux system that already has `wget` installed, FUSE enabled in the kernel, and a Mycelium connectivity. If you deployed a VM on the ThreeFold Grid according to the instructions above, then these requirements are already met.
+
+### Install Binaries
+
+The three binary executables needed to operate QSFS are provided in statically compiled form with no dependencies. You can download them from GitHub according to the links on each project's release page:
+
+- 0-db-fs: [https://github.com/threefoldtech/0-db-fs/releases](https://github.com/threefoldtech/0-db-fs/releases)
+- 0-db: [https://github.com/threefoldtech/0-db/releases](https://github.com/threefoldtech/0-db/releases)
+- 0-stor: [https://github.com/threefoldtech/0-stor_v2/releases](https://github.com/threefoldtech/0-stor_v2/releases)
+
+
+Here is an example with the latest versions of each component at the time of publishing this guide. We will also download a hook script that is the final needed component:
+
+```
+wget -O /usr/local/bin/zdbfs https://github.com/threefoldtech/0-db-fs/releases/download/v0.1.11/zdbfs-0.1.11-amd64-linux-static
+wget -O /usr/local/bin/zdb https://github.com/threefoldtech/0-db/releases/download/v2.0.8/zdb-2.0.8-linux-amd64-static
+wget -O /bin/zstor https://github.com/threefoldtech/0-stor_v2/releases/download/v0.4.0/zstor_v2-x86_64-linux-musl
+wget -O /usr/local/bin/zdb-hook.sh https://raw.githubusercontent.com/threefoldtech/quantum-storage/master/lib/zdb-hook.sh
+
+# Make them all executable
+chmod +x /usr/local/bin/* /bin/zstor
+```
+
+One note here is that the name and location of the `zstor` executable must match what is shown here for the hook script to work properly.
+
+### Directories
+
+Two directories will be needed for QSFS operation. You can create them as follows:
+
+```sh
+mkdir -p /mnt/qsfs
+mkdir -p /data
```

Comment:
What is the use of this directory?

Reply:
I've explained what each is used for, along with adding a note that they can be changed.